build.gradle

```diff
@@ -11,6 +11,7 @@ apply plugin: 'com.diffplug.spotless'
 apply from: 'formatter/formatting.gradle'
 // for javadocs and checks spotless doesn't do
 apply plugin: 'checkstyle'
+checkstyleTest.enabled = false
```

Comment:
The checkstyle failures were because you are using star imports, which we intentionally forbid for [Reasons](https://www.javadude.com/posts/20040522-import-on-demand-is-evil/).

Reply:
You can suppress test path in the config, see https://github.com/opensearch-project/opensearch-ai-flow-framework/blob/main/config/checkstyle/checkstyle_suppressions.xml where javadocs are suppressed rather than turning the whole thing off.  If we really want to allow * in test we could add it there.

But with modern IDEs I really don't think it's necessary. When writing code I put the * in there, and then let my IDE sort out what to actually import.

Reply:
I'm also fine with you turning it off for draft while you develop and fixing it at the end, I recognize this PR is a work in progress!